src/main/java/mate/academy/onlinebookstore/model/CartItem.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Data
+@Entity
+@SQLDelete(sql = "UPDATE cart_item SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")
```

Comment:
```suggestion
@SQLRestriction("is_deleted=false")
```
@Where is deprecated